### ISSUE TITLE
help me please

### ISSUE THREAD

**Issue by Scrooner13054** [DeveloperRic/CustomShop#22]
whenever I type /shop or /customshop:shop nothing happens but I have all the plugins in and the command auto fill what do I do? even /shop reload doesn't work

**Comment by DeveloperRic** [comment]
Hi, please can you attach the server messages prefixed with `[CustomShop]`.

These messages are typically printed when your start your server, reload plugins, or use any `/shop` command.

Please also attach any error stacktraces

**Comment by Scrooner13054** [comment]
<img width="958" alt="Screenshot 2021-11-18 at 12 07 00 PM" src="https://user-images.githubusercontent.com/94638258/142489554-b8cb3eab-01bd-476c-885c-724878475b70.png">
this?

**Comment by DeveloperRic** [comment]
No, copy & paste the lines in the server console that start with [CustomShop]

Also copy any lines that say "error" or "exception"

If what you paste into your reply is very long, put it in a .txt file & upload that instead

**Comment by Scrooner13054** [comment]
none of them start with custom shop

**Comment by DeveloperRic** [comment]
That should never happen. What does it say when u type /help CustomShop

Also try copying your entire server log into a text file & ctrl+f for CustomShop

**Comment by Scrooner13054** [comment]
it says nothibg

**Comment by DeveloperRic** [comment]
What is the **exact** output

Also please attach your server log as a text file
